lib/commands/deploynodeapp.js

```diff
@@ -72,6 +78,35 @@ var descriptor = defaults.defaultDescriptor({
     name: 'Preserve policies from previous revision',
     shortOption: 'P',
     toggle: true
+  },
+  runtime: {
+    name: 'Runtime to deploy the Node.js app to',
+    shortOption: 'B',
+    required: false
+  },
+  stream: {
+    name: 'Stream EdgeServices API calls',
+    shortOption: 's',
+    toggle: true
+  },
+  'node-version': {
+    name: 'Version of node to use in EdgeServices',
+    shortOption: 'x',
```

Comment:
I get ya. I think if the short flag is not relevant to the long flag than we probably just leave it out. Especially for something like node-version that probably wont get set in all cases.